packages/headless/src/features/commerce/common/actions.ts

```diff
@@ -49,19 +51,32 @@ export const buildCommerceAPIRequest = (
 export const buildBaseCommerceAPIRequest = (
   state: StateNeededByQueryCommerceAPI,
   relay: Relay,
+  navigatorContext: NavigatorContext,
   slotId?: string
 ): BaseCommerceAPIRequest => {
-  const {view, user, ...restOfContext} = state.commerceContext;
+  const {view, ...restOfContext} = state.commerceContext;
+  const meta = relay.getMeta('');
```

Comment:
I'm not against this, but this seems to be the only place where we're doing this (usually we just do `clientId: relay.getMeta('').clientId`.

Reply:
I originally did this because I was re-using it for the other properties. I'll inline it!

Reply:
Done in https://github.com/coveo/ui-kit/pull/4113/commits/4ac9380f5842bd413277a8f55bc48b5c3dd0d90f